modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-no-slip-outflow-bcs.i

```diff
@@ -0,0 +1,182 @@
+mu=1.1
+rho=1.1
+advected_interp_method='average'
+velocity_interp_method='rc'
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 10
+    ymin = -1
+    ymax = 1
+    nx = 100
+    ny = 20
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  fv_bcs_integrity_check = true
+[]
+
+[Variables]
+  [u]
+    type = INSFVVelocityVariable
+    initial_condition = 1
+  []
+  [v]
+    type = INSFVVelocityVariable
+    initial_condition = 1
+  []
+  [pressure]
+    type = INSFVPressureVariable
+  []
+[]
+
+[FVKernels]
+  [mass]
+    type = INSFVMassAdvection
+    variable = pressure
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    vel = 'velocity'
+    pressure = pressure
+    u = u
+    v = v
+    mu = ${mu}
+    rho = ${rho}
+  []
+
+  [u_advection]
+    type = INSFVMomentumAdvection
+    variable = u
+    advected_quantity = 'rhou'
+    vel = 'velocity'
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    pressure = pressure
+    u = u
+    v = v
+    mu = ${mu}
+    rho = ${rho}
+  []
+  [u_viscosity]
+    type = FVDiffusion
+    variable = u
+    coeff = ${mu}
+  []
+  [u_pressure]
+    type = INSFVMomentumPressure
+    variable = u
+    momentum_component = 'x'
+    p = pressure
+  []
+
+  [v_advection]
+    type = INSFVMomentumAdvection
+    variable = v
+    advected_quantity = 'rhov'
+    vel = 'velocity'
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    pressure = pressure
+    u = u
+    v = v
+    mu = ${mu}
+    rho = ${rho}
+  []
+  [v_viscosity]
+    type = FVDiffusion
+    variable = v
+    coeff = ${mu}
+  []
+  [v_pressure]
+    type = INSFVMomentumPressure
+    variable = v
+    momentum_component = 'y'
+    p = pressure
+  []
+[]
+
+[FVBCs]
+  [inlet-u]
+    type = INSFVInletVelocityBC
+    boundary = 'left'
+    variable = u
+    function = '1'
+  []
+  [inlet-v]
+    type = INSFVInletVelocityBC
+    boundary = 'left'
+    variable = v
+    function = '0'
+  []
+  [walls-u]
+    type = INSFVNoSlipWallBC
+    boundary = 'top bottom'
+    variable = u
+    function = 0
+  []
+  [walls-v]
+    type = INSFVNoSlipWallBC
+    boundary = 'top bottom'
+    variable = v
+    function = 0
+  []
+  [outlet_u]
+    type = INSFVMomentumAdvectionOutflowBC
+    variable = u
+    advected_quantity = 'rhou'
+    vel = 'velocity'
+    advected_interp_method = ${advected_interp_method}
+    u = u
+    v = v
+    boundary = 'right'
+  []
+  [outlet_v]
+    type = INSFVMomentumAdvectionOutflowBC
+    variable = v
+    advected_quantity = 'rhov'
+    vel = 'velocity'
+    advected_interp_method = ${advected_interp_method}
+    u = u
+    v = v
+    boundary = 'right'
+  []
```

Comment:
so these do nothing in this test right? By that I mean that they change nothing
They have the same contribution as the advection kernel and they do the same fdf tagging as the pressure BC

Reply:
correct and correct